test/MPI/test_MPI_numpy.py

```diff
@@ -16,17 +17,18 @@
 @mark_mpi
 @dtype_loop
 @lshape_loop  # enables local domain
-def test_MPI_numpy_load_comm(tempdir_MPI, dtype, lshape):
+@parallel_format_loop
+def test_MPI_load_comm(tempdir_MPI, dtype, lshape, format):
 
     comm = get_comm()
     rank = comm.rank
-    ftmp = tempdir_MPI + "/foo_numpy_mpiio_load_comm.npy"
+    ftmp = tempdir_MPI + "/mpiio_load_comm"
+    if format == "hdf5":
+        ftmp += ".h5/data"
 
-    mult = tuple(comm.size if i == 0 else 1 for i in range(len(lshape)))
-    write_global_array(comm, ftmp, lshape, dtype=dtype, mult=mult)
-    global_array = numpy.load(ftmp)
-
-    local_array = io.load(ftmp, comm=comm)
+    write_global_array(comm, ftmp, lshape, dtype=dtype, format=format)
+    global_array = io.load(ftmp, format=format)
+    local_array = io.load(ftmp, comm=comm, format=format)
```

Comment:
This bit here also kind of tests the default naming of parallel hdf5  [tests](https://github.com/Lyncs-API/lyncs.io/blob/835136c51ee95fbb54786a63cd050820dcec182e/test/MPI/test_MPI_hdf5.py#L93) in Lines 93-98 which we might want to remove/skip?

Reply:
yeah, wasn't sure whether to remove it or not.. For now better more than less.
I would like something similar also for Dask but not now. :)